delocate/delocating.py

```diff
@@ -266,6 +385,10 @@ def delocate_path(tree_path, lib_path,
         Default is callable rejecting only libraries beginning with
         ``/usr/lib`` or ``/System``.  None means copy all libraries. This will
         usually end up copying large parts of the system run-time.
+    executable_path : str, optional
+        An alternative path to use for resolving `@executable_path`.
+    ignore_missing : bool, default=False
+        Continue even if missing dependencies are detected.
```

Comment:
```suggestion

    executable_path : None or str, optional
        If not None, an alternative path to use for resolving `@executable_path`.
    ignore_missing : {False, True}, optional
        Continue even if missing dependencies are detected.
```

Reply:
`bool, default=False` is less ambiguous than `{False, True}, optional` and seemed valid according to this style guide:  https://numpydoc.readthedocs.io/en/latest/format.html#parameters

Reply:
Aha - the recommendation changed since the old days.   That's fine.